jdaviz/configs/imviz/helper.py

```diff
@@ -26,6 +30,71 @@ def load_data(self, data, parser_reference=None, **kwargs):
 
         self.app.load_data(filepath, parser_reference=parser_reference, **kwargs)
 
+    def load_static_regions(self, regions, data_label, **kwargs):
+        """Load given region(s) into the viewer.
+        Once loaded, the region(s) cannot be modified.
+
+        Parameters
+        ----------
+        regions : dict
+            Dictionary mapping desired region name to one of the following:
+
+            * Astropy ``regions`` object
+            * ``photutils`` apertures (limited support until ``photutils``
+              fully supports ``regions``)
+            * Numpy boolean array (shape must match data)
+
+            Region name that starts with "Subset" is forbidden and reserved
+            for internal use only.
+
+        data_label : str
+            Label to retrieve a specific data set from the viewer instance.
+            Glue Subset object representing this region will be created
+            based on this data set.
+
+        kwargs : dict
+            Extra keywords to be passed into the region's ``to_mask`` method.
+            This is ignored if Numpy array is given.
+
+        """
+        # TODO: Refactor after https://github.com/spacetelescope/jdaviz/pull/644
+        # is merged.
+        # Or do I have to use self.app.get_data_from_viewer for some reason?
+        data = self.app.data_collection[
+            self.app.data_collection.labels.index(data_label)]
```

Comment:
@astrofrog , is there any reason not to call `data_collection` directly? I don't see any unintended consequences but you know better.

Reply:
Note to self: I wonder if linking problem can be deferred if I use the "reference image" and take away `data_label` option...